server-config.rwm.json

```diff
@@ -0,0 +1,16 @@
+{
+    "url": "https://relay.rif-wallet-services.mainnet.rifcomputing.net",
+    "port": 8090,
+    "relayHubAddress": "0xAd525463961399793f8716b0D85133ff7503a7C2",
+    "relayVerifierAddress": "0xB86c972Ff212838C4c396199B27a0DBe45560df8",
+    "deployVerifierAddress": "0xc67f193Bb1D64F13FD49E2da6586a2F417e56b16",
+    "feesReceiver": "0x52D107bB12d83EbCBFb4A6Ad0ec866Bb69FdB5Db",
```

Comment:
could you please remove the `feesReceiver`? I know that this configuration isn't the final one, but better not to include it, since it won't be there.

Reply:
yeah sure. I'll send you variables to add for this environment via Slack